interview-form.html

```diff
@@ -0,0 +1,173 @@
+          <input placeholder="Enter your Github User URL" type="text" name="github" id="github-input">
+        </span>
+        <div class="error-text">The github user doesn't exist</div>
+      </div>
+
+      <div id="stream" class="form-input-group">
+        <label for="stream-input">Stream have in 12th</label>
+        <span class="text-input">
+          <input placeholder="Enter your Stream,ex.PCM+CS" type="text" name="stream" id="stream-input">
+        </span>
+        <div class="error-text">This can't be empty</div>
+      </div>
+      
+      <hr class="separator">
+
+      <div id="languages" class="form-input-group">
+        <label for="languages-input">Languages</label>
+        <div class="option-group">
+          <input type="checkbox" name="languages" id="languages-swift" value="swift">
+          <label for="languages-swift">Swift</label>
+          <input type="checkbox" name="languages" id="languages-typescript" value="typescript">
+          <label for="languages-typescript">Typescript</label>
+          <input type="checkbox" name="languages" id="languages-kotlin" value="kotlin">
+          <label for="languages-kotlin">Kotlin</label>
+          <input type="checkbox" name="languages" id="languages-golang" value="golang">
+          <label for="languages-golang">Go</label>
+          <input type="checkbox" name="languages" id="languages-c#" value="c#">
+          <label for="languages-c#">C#</label>
+          <input type="checkbox" name="languages" id="languages-javascript" value="javascript">
+          <label for="languages-javascript">JavaScript</label>
+          <input type="checkbox" name="languages" id="languages-java" value="java">
+          <label for="languages-java">Java</label>
+          <input type="checkbox" name="languages" id="languages-python" value="pyhton">
+          <label for="languages-python">Python</label>
+        </div>
+        <div class="error-text">Choose at least one language</div>
+      </div>
+
+
+      <div id="frameworks" class="form-input-group">
+        <label for="frameworks-input">Frameworks</label>
+        <div class="option-group">
+          <input type="checkbox" name="frameworks" id="frameworks-quick" value="quick">
+          <label for="frameworks-quick">Quick</label>
+          <input type="checkbox" name="frameworks" id="frameworks-vapor" value="vapor">
+          <label for="frameworks-vapor">Vapor</label>
+          <input type="checkbox" name="frameworks" id="frameworks-javalin" value="javalin">
+          <label for="frameworks-javalin">Javalin</label>
+          <input type="checkbox" name="frameworks" id="frameworks-spark" value="spark">
+          <label for="frameworks-spark">Spark</label>
+          <input type="checkbox" name="frameworks" id="frameworks-angular" value="angular">
+          <label for="frameworks-angular">Angular</label>
+          <input type="checkbox" name="frameworks" id="frameworks-vue.js" value="vue.js">

```

Comment:
By Convention, we write JS libraries as "NameJS". So for vue it would be "VueJS".
This of course conflicts with how they write it themselves, but it will let us be consistent.
Consider a similar change for "ReactJS"